assignment3/css/index.css

```diff
@@ -0,0 +1,93 @@
+* body {
+	margin:0px;
+	padding:0px;
+	background-color: #f2f2f2;
+}
+.container {
+	height:auto;
+	width:80%;
+	margin:0px auto;
+}
+.header {
+	background-color: white;
+}
+.navbar {
+	 width:100%;
+	 height:60px;
+}
+img {
+	height:50px;
```

Comment:
@bhartijain Add either height or width for the image. If you have both height and width it will stretch.